### ISSUE TITLE
Fix `monitor` method docsting and UT

### ISSUE THREAD

**Issue by arbulu89** [SUSE/SAPStartSrv-resourceAgent#11]
Follow up item for: https://github.com/SUSE/SAPStartSrv-resourceAgent/pull/10

1. Unify docstring
2. Reach 100% test coverage

**Comment by fmherschel** [comment]
@arbulu89 Could you explain why the test coverage is now les then 100%? Is it, because I did add an if-then-else and both twiggs needs to be called in the test? I just like to get that point :)

**Comment by arbulu89** [comment]
> @arbulu89 Could you explain why the test coverage is now les then 100%? Is it, because I did add an if-then-else and both twiggs needs to be called in the test? I just like to get that point :)

Exactly. With the new code, one of the `if/else` statements is not covered. The old test just goes through 1 path
